deepchecks/tabular/__init__.py

```diff
@@ -0,0 +1,37 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Package for tabular functionality."""
+from .dataset import Dataset
+from .base import (
+    TabularContext,
+    TabularCheck,
+    Suite,
```

Comment:
I think if leave it "Suite" and not "TabularSuite" we can do the same for the rest of classes

Reply:
like this?
- `TabularContext` -> `Context`
- `TabularCheck` -> `Check`

do not know/ @ItayGabbay , @noamzbr  what do you think?

Reply:
I think it's better

Reply:
renamed